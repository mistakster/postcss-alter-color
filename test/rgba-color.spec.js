const process = require('./utils/process');

it('should alter rgba colors', () => {
  const source = `
div {
  color: rgba(0,0,0,1);
}
`;

  return process(source, {from: 'black', to: 'red'})
    .andMatchSnapshot();
});

it('should alter rgba color in a complex value', () => {
  const source = `
div {
  color: rgba(0,0,0,1);
  border: 1px solid rgba(255,255,255,1);
  outline: 1px solid rgba(0,0,0,1);
}
`;

  return process(source, {from: 'black', to: 'red'})
    .andMatchSnapshot();
});

it('should alter rgba color and opacity in a complex value', () => {
  const source = `
div {
  color: rgba(0,0,0,0.5);
  border: 1px solid rgba(255,255,255,1);
  outline: 1px solid rgba(0,0,0,1);
}
`;

  return process(source, {from: 'black', to: 'red'})
    .andMatchSnapshot();
});
