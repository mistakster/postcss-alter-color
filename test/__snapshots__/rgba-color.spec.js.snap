// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should alter rgba color and opacity in a complex value 1`] = `
"
div {
  color: rgba(0,0,0,0.5);
  border: 1px solid rgba(255,255,255,1);
  outline: 1px solid rgba(255,0,0,1);
}
"
`;

exports[`should alter rgba color in a complex value 1`] = `
"
div {
  color: rgba(255,0,0,1);
  border: 1px solid rgba(255,255,255,1);
  outline: 1px solid rgba(255,0,0,1);
}
"
`;

exports[`should alter rgba colors 1`] = `
"
div {
  color: rgba(255,0,0,1);
}
"
`;
